hugo/lib/getArticles.js

```diff
@@ -81,6 +81,7 @@ Promise.coroutine(function*() {
     });
   } catch (err) {
     console.error(err);
+    process.exit(6);
```

Comment:
I saw it being used on a Linux forum for p2p connection errors. But I only found one other source using 6 for connection errors. I am good switching all to 1.